README.md

```diff
@@ -0,0 +1,23 @@
+Syditest
+========
+
+Syditest is an integrations testing system for Matrix Identity servers, similar
+to sytest (although using python rather than perl).
+
+Syditest is a standard set of python unit tests, although is intended to be run
+with Twisted's Trial.
+
+To launch the IS server, it attempts to import
+`syditest_subject.launcher.SyditestLauncher`. This must be provided separately
```

Comment:
Mostly that in the the subsequent versions it's now launching multiple copies of the IS with different parameters (eg. with & without terms) so it gives the test framework more flexibility and hopefully simplifies what the testee has to provide, ie. a function that launches an  IS with given parameters rather than, "you must have launches one IS like this on this port, one with the config on this port" and so on.

Reply:
That should be the case now? The launcher can run whatever binary it needs to in whatever environment it needs to.

Reply:
so how do I get it to run sydent in its own virtualenv?

Reply:
Easiest way would probably be to have sydent's launcher run the python binary from the virtualenv in question.

Reply:
conclusion: add a SYDENT_PYTHON env var that you can set up to point to the python binary you want to use to run sydent

Reply:
Done in https://github.com/matrix-org/sydent/pull/179/commits/10d148672451215c19773e79b03ec843cea1d656